repo.go

```diff
@@ -55,9 +55,12 @@ type LocalStore interface {
 	// GetSigners return a list of signers for a role.
 	GetSigners(string) ([]keys.Signer, error)
 
-	// SavePrivateKey adds a signer to a role.
+	// SaveSigner adds a signer to a role.
 	SaveSigner(string, keys.Signer) error
 
+	// ChangePassphrase changes the passphrase for a role keys file.
+	ChangePassphrase(string) error
```

Comment:
@ethan-lowman-dd: I'm open to suggestions on that one too 👍 I was thinking to do it in a separate function similar to how `getPassphrase` is done but thought this passes fewer variables back and forth. 

Anyway, can you elaborate a bit more on your proposal above? If `ChangePassphrase` won't be part of `LocalStore`, why should I do the assertion part or do I get this wrong?

Reply:
This was the idea: https://github.com/theupdateframework/go-tuf/commit/4817580cf7e910a17d1ce6aada2371134984cf96

If you like it, feel free to cherry pick.

Reply:
Ah, neat! I haven't seen this before and so I assumed I should be able to call `r.local.ChangePassphrase()` in case of successful type assertion (which failed of course). But this is nice, thanks for the tip! 👍